apps/heft/src/utilities/Async.ts

```diff
@@ -3,7 +3,13 @@
 
 import { ScopedLogger } from '../pluginFramework/logging/ScopedLogger';
 
+/**
+ * Helpful async utility methods
+ */
 export class Async {
```

Comment:
Perhaps this should be better formalized and moved to `node-core-library`? @octogonz?

Reply:
@octogonz and @iclanton  Before I forget, I can redo this PR to just have the staticAssetsToCopy change.

But I really think we should have a `heft-plugin-utils` project on here with methods that are useful when creating plugins for heft. I know we've copied the Async static methods from here into 2 other internal repos, and I think it would be best if we found a way to consolidate that code into one place.